datajunction-server/datajunction_server/api/sql.py

```diff
@@ -280,6 +300,7 @@ async def get_node_sql(  # pylint: disable=too-many-locals
     current_user: User,
     validate_access: access.ValidateAccessFn,  # pylint: disable=redefined-outer-name
     background_tasks: BackgroundTasks,
+    ignore_errors: bool = True,
```

Comment:
Why is ignoring errors the default value?

Reply:
We sometimes still get the type inference errors, which I generally think we should ignore. If the generated query doesn't run, the user will know anyway.